### PR TITLE
document warmup options

### PR DESCRIPTION
**Pull request by kalantar**
document changes implemented in https://github.com/iter8-tools/hub/pull/56 for https://github.com/iter8-tools/iter8/issues/1406

### CODE REVIEW

File: docs/user-guide/tasks/grpc.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ In addition, the following fields are defined by this task.
 | dataURL  | string (URL) | URL where JSON data to be used in call requests is located. |
 | binaryDataURL  | string (URL) | URL where binary data to be used in call requests is located. |
 | metadataURL  | string (URL) | URL where the JSON metadata data to be used in call requests is located. |
+| warmupNumRequests | int | Number of requests to be sent in a warmup task (results are ignored).  |
```
Comment (Alan-Cha) [inline docs/user-guide/tasks/grpc.md:36]:
```suggestion
| warmupNumRequests | int | Number of requests to be sent in a warmup task (results are ignored). |
```

File: docs/user-guide/tasks/grpc.md [hunk 2]
```diff
@@ -33,6 +33,8 @@ In addition, the following fields are defined by this task.
...
+| warmupDuration | string | Duration of warmup task (results are ignored). Specified in the [Go duration string format](https://pkg.go.dev/maze.io/x/duration#ParseDuration) (example, 5s). If both warmupDuration and warmupNumRequests are specified, then warmupDuration is ignored. |
```
Comment (Alan-Cha) [inline docs/user-guide/tasks/grpc.md:37]:
```suggestion
| warmupDuration | string | Duration of warmup task (results are ignored). Specified in the [Go duration string format](https://pkg.go.dev/maze.io/x/duration#ParseDuration) (example: `5s`). If both warmupDuration and warmupNumRequests are specified, then warmupDuration is ignored. |
```

File: docs/user-guide/tasks/http.md [hunk 3]
```diff
@@ -32,6 +32,8 @@ iter8 k launch \
 | payloadURL  | string (URL) | URL from which to download the content that will be used as the request payload. If this field is specified, Iter8 will send HTTP POST requests to the app using this content as the payload. |
 | payloadStr  | string | String data to be used as the request payload. If this field is specified, Iter8 will send HTTP POST requests to the app using this string as the payload. |
 | contentType  | string | Content type of the payload. This is intended to be used in conjunction with one of the `payload*` fields. If this field is specified, Iter8 will send HTTP POST requests to the app using this as the Content-Type header value. |
+| warmupNumRequests | int | Number of requests to be sent in a warmup task (results are ignored). |
+| warmupDuration | string | Duration of warmup task (results are ignored). Specified in the [Go duration string format](https://pkg.go.dev/maze.io/x/duration#ParseDuration) (example, 5s). If both warmupDuration and warmupNumRequests are specified, then warmupDuration is ignored. |
```
Comment (Alan-Cha) [inline docs/user-guide/tasks/http.md:36]: See other comment

File: docs/user-guide/tasks/grpc.md (hunk 2 above)
Comment (sriumcp) [reply docs/user-guide/tasks/grpc.md:37]: Yes, but we need to consistently do this for other fields also.

**Review by sriumcp — APPROVED** [review]
lgtm

**Pull request merged**